src/Stryker.Core/Stryker.Core/Reporters/MarkdownSummaryReporter.cs

```diff
@@ -0,0 +1,133 @@
+            // This reporter does not report during the testrun
+        }
+
+        public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
+        {
+            var files = reportComponent.GetAllFiles();
+            if (files.Any())
+            {
+                var filename = _options.ReportFileName + ".md";
+                var reportPath = Path.Combine(_options.ReportPath, filename);
+                var reportUri = "file://" + reportPath.Replace("\\", "/");
+
+                GenerateMarkdownReport(_options, reportPath, files, reportComponent.GetMutationScore());
+
+                _console.WriteLine();
+                _console.MarkupLine("[Green]Your Markdown summary has been generated at:[/]");
+
+                if (_console.Profile.Capabilities.Links)
+                {
+                    // We must print the report path as the link text because on some terminals links might be supported but not actually clickable: https://github.com/spectreconsole/spectre.console/issues/764
+                    _console.MarkupLineInterpolated($"[Green][link={reportUri}]{reportPath}[/][/]");
+                }
+                else
+                {
+                    _console.MarkupLineInterpolated($"[Green]{reportUri}[/]");
+                }
+            }
+        }
+
+        private void GenerateMarkdownReport(StrykerOptions options, string reportPath, IEnumerable<IFileLeaf> files, double mutationScore)
+        {
+            if (!files.Any())
+            {
+                return;
+            }
+
+            var mdSummaryDocument = new MdDocument();
+            mdSummaryDocument.Root.Add(new MdHeading(1, "Mutation Testing Summary"));
+
+            var mdSummary = new MdTable(new MdTableRow(new[] { "File", "% score", "# killed", "# timeout", "# survived", "# no cov", "# error" }));
+
+            foreach (var file in files)
+            {
+                mdSummary.Add(GenerateFileData(file));
+            }
+
+            mdSummaryDocument.Root.Add(mdSummary);
+            mdSummaryDocument.Root.Add(new MdHeading(2, new MdTextSpan($"The final mutation score is {mutationScore * 100:N2}%")));
+            mdSummaryDocument.Root.Add(new MdHeading(3, new MdEmphasisSpan($"Coverage Thresholds: high:{_options.Thresholds.High} low:{_options.Thresholds.Low} break:{_options.Thresholds.Break}")));
+
+            _fileSystem.Directory.CreateDirectory(Path.GetDirectoryName(reportPath));
+            _fileSystem.Directory.CreateDirectory(Path.GetDirectoryName(reportPath));
+            using var mdFile = _fileSystem.File.Create(reportPath);
+            mdSummaryDocument.Save(mdFile);
+        }
+
+        private MdTableRow GenerateFileData(IFileLeaf fileScores)
+        {
+            var values = new List<string>();
+
+            values.Add(fileScores.RelativePath ?? "All files");
+
+            var mutationScore = fileScores.GetMutationScore();
+
+            if (fileScores.IsComponentExcluded(_options.Mutate))
```

Comment:
You can remove this, excluding is way more granular than only on files these days, so we should no longer mark files as excluded but rather show the amount of ignored mutants in a file. The file will then display a score of N/A instead.